### ISSUE TITLE
radiocells.org web site Openstreetmap tower data issues

### ISSUE THREAD

**Issue by gdt** [openbmap/android#149]
On radiocells.org, there is a leaflet layer switcher and one can enable "Openstreetmap tower data".  There are a few issues with this:
 * It's not documented (that I can find) what tags are used.  I can read the js, but that seems not the right place to document.
 * The list of tags seems not entirely consistent.   In particular man_made=tower seems deprecated for many things, and the real question is whether man_made=mast/tower:type=communications is sufficient to show, or if one needs a tag specifically about cell.   I lean to showing all communications towers and all cell antennas (as would be on buildings) as the useful choice.
 * I can't see any of my local towers.  For example, see <https://radiocells.org/cell_provider/1067?lat=42.33174949991875&lon=-71.67233169078827&zoom=18> vs <https://www.openstreetmap.org/#map=18/42.33174/-71.67425>
 * It's really hard to tell, but it feels like checking the OSM tower box breaks loading/displaying of cells.  But now I think that display of cells is broken at high zooms (13 ok, 14 broken?) always.  This used to be ok.

**Comment by wish7code** [comment]
Hey Greg,  thanks for reporting!

You're right, unfortunately it's not yet documented, but please have a look at http://overpass-turbo.eu/s/fXa . This will show our current query.

So in this specific case, the masts weren't displayed due to the missing communication:mobile_phone=yes. Suggested fix: include tower:type=communication as in **http://overpass-turbo.eu/s/fXd**

Any other ideas highly welcome..

>  I lean to showing all communications towers and all cell antennas (as would be on buildings) as the useful choice.

I would agree: from my experience the OSM tagging is very inconsistent for cell towers and poorly documented too. Thus I would rather prefer to include some 'false positives' instead of missing some towers. E.g. a very common error I've seen is the misuse of communications_tower instead of man_made=mast

Cheers
Toby

**Comment by gdt** [comment]
It looks like "tower:type"="communication" is searched for.   So given that, some of the other queries are redundant.  Then there is  "communication:mobile_phone"="yes", which gets anything tagged as that.   The next question is whether man_made=antenna needs to be searched in addition, and I'd tend to yes.     I agree the tagging is messy.  Part of the issue is that it's hard to tell looking at it, part that people don't bother with the details, and part that even people who care about this don't really know the tagging rules.

With your query, I do see the towers (masts in en_GB) that I had tagged, and I also see a man_made=tower tower_type=communication that I tagged long ago (should be mast, for a police department), but has never shown up on the website.   So I think there may still be something wrong.

If I check the "show OSM tower data box", should I then see some visible (and clickable) items on the map?  Does that work for you?

**Comment by wish7code** [comment]
> If I check the "show OSM tower data box", should I then see some visible (and clickable) items on the map? Does that work for you?

Nop, but fixed by now :-)

**Comment by gdt** [comment]
It's working much better.  I see two issues (with some possibility of confusion on my part):
 * the icon seems to have the upper left point on the tower, rather than than the middle, which is confusing because at many zoom levels it looks like it's on the wrong side of the road
 * I panned around after loading towers, and didn't see the new ones.   I waited a while.   I then unchecked and rechecked the show towers box, and then after a bit they loaded.

But things are hugely better - thanks for fixing.
